Error: Cannot find module 'fs-extra'

`fs-extra` is used by `next-pwa` but not actually in the dependencies. This makes the library totally unusable for anyone using

- A package manager that uses PnP
- Yarn 2
- Anyone who isn't coincidentally installing a package that properly depends on `fs-extra`

```
$ next build

> Build error occurred
Error: Cannot find module 'fs-extra'
Require stack:
- /usr/src/project/node_modules/next-pwa/index.js
- /usr/src/project/next.config.js
- /usr/src/project/node_modules/next/dist/next-server/server/config.js
- /usr/src/project/node_modules/next/dist/build/index.js
- /usr/src/project/node_modules/next/dist/cli/next-build.js
- /usr/src/project/node_modules/next/dist/bin/next
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/usr/src/project/node_modules/next-pwa/index.js:4:12)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/src/project/node_modules/next-pwa/index.js',
    '/usr/src/project/next.config.js',
    '/usr/src/project/node_modules/next/dist/next-server/server/config.js',
    '/usr/src/project/node_modules/next/dist/build/index.js',
    '/usr/src/project/node_modules/next/dist/cli/next-build.js',
    '/usr/src/project/node_modules/next/dist/bin/next'
  ]
}
```

--- a/index.js
+++ b/index.js
@@ -1,7 +1,7 @@
 'use strict'
 
 const path = require('path')
-const fs = require('fs-extra')
+const fs = require('fs')
 const globby = require('globby')
 const crypto = require('crypto')
 const { CleanWebpackPlugin } = require('clean-webpack-plugin')

